The application 'temprxGO.py' lost its connection to the display localhost:10.0;
most likely the X server was shut down or you killed/destroyed
the application.
